src/sagemaker/serve/builder/model_builder.py

```diff
@@ -577,12 +580,19 @@ def build(
         )
 
         self.serve_settings = self._get_serve_setting()
+
+        hf_model_md = get_huggingface_model_metadata(self.model,
```

Comment:
https://huggingface.co/docs/sagemaker/reference#inference-toolkit-environment-variables I thought it was `HF_API_TOKEN `

Reply:
Is there a case where the HF_API_TOKEN can be set, but the HUGGING_FACE_HUB_TOKEN is empty/null?